store/csv.go

```
@@ -185,12 +226,14 @@ func (store *CsvStore) LoadAll() (samples []experiment.Experiment, err error) {
 
 	samples = make([]experiment.Experiment, 0)
 	for _, f := range files {
-		base := strings.Split(f.Name(), ".")[0]
-		name := strings.SplitN(base, "-", 2)[1]
-		if len(name) > 0 {
-			loaded, err := store.load(f.Name(), name)
-			if err == nil {
-				samples = append(samples, loaded)
+		if filepath.Ext(f.Name()) == ".csv" {
```

so if the file doesn't end in .csv it silently fails? is this what we want and is there a test I missed for this somewhere?